src/STRPAK.CPP

```diff
@@ -1181,10 +1182,10 @@ char* strCase( char* s, const char toCases[3])
 		else				// spaces don't need case conversion
 		{
 			if (itc[which]==upper)
-				CharUpperBuff( s+i, 1);
+				*((unsigned char*)s+i) = toupper(*((unsigned char*)s + i));
```

Comment:
I want to make sure we understand what this code is doing. I'm not sure I do yet.